_P208_Nokia_LCD_5110.ino

```diff
@@ -271,12 +277,19 @@ boolean displayText3(char deviceTemplate[][48], struct EventStruct *event ){ //4
             lcd3->setTextSize(1);
           }
           String tmpString = deviceTemplate[x];
-          String newString = "";
+          String newString = " ";
+          String newString3 = " ";

```

Comment:
All constant strings should best be put in flash memory using the `F()` macro. so this should be `String newString = F(" ");` (that's applicable in other places too).

Reply:
Why should constant strings best be put in memory?

Reply:
They should be in **flash** memory instead of (ram) memory, as ram is more scarce than flash, and it already has to be in flash anyway, that's all managed by that `F()` macro 😃

Reply:
In the next commit, F() is used.